src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,34 +40,14 @@
 import frc.robot.commands.autoSequences.sideThreePieceRed;
 import frc.robot.commands.autoSequences.sideTwoPieceBlue;
 import frc.robot.commands.autoSequences.sideTwoPieceRed;
+import frc.robot.pathplanner.CommandChooser;
 import frc.robot.subsystems.AnglerPIDSubsystem;
 import frc.robot.subsystems.ClimberSubsystem;
 import frc.robot.subsystems.DrivebaseSubsystem;
 import frc.robot.subsystems.IntakeSubsystem;
 import frc.robot.subsystems.LedSubsystem;
 import frc.robot.subsystems.ShooterSubsystem;
 
-enum AutoOption {
```

Comment:
This was converted to String so that we could add options dynamically. Unfortunately, this does open up the possibility of making a typo and the compiler will no longer catch it. Any other options?